### PR TITLE
add Chinese translation

### PR DESCRIPTION
**Pull request by Sharuru**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The localization process is defined below:
 - Japanese: [@kaakaa](https://github.com/kaakaa)
 - Polish: [@burasuk](https://github.com/burasuk)
 - Spanish: [@ernestoittig](https://github.com/ernestoittig)
+- Chinese: [@sharuru](https://github.com/sharuru)
```
Comment (kaakaa) [inline CONTRIBUTING.md:30]: There are two kinds of Chinese (`zh-CN` and `zh-TW`) in Mattermost, so I think that it has better to clarify which Chinese have been translated. According to [Mattermost docs](https://docs.mattermost.com/developer/localization.html#translation-maintenance), `zh-CN` is called `Simplified Chinese`, so what do you think about rewriting this and README to `Simplified Chinese`?

**Review by kaakaa — COMMENTED** [review]
Thanks @Sharuru! Your translations works fine. But, I have a question about documentation.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Sharuru) [reply CONTRIBUTING.md:30]: Oh, it's my mistake.
`Simplified Chinese` would be better.

**Review by kaakaa — APPROVED** [review]
LGTM 👍 

@hanzei Can I merge this?

**Review by hanzei — APPROVED** [review]

**Pull request merged**